app/screens/settings-screen/username-screen.tsx

```diff
@@ -130,8 +118,10 @@ export const UsernameScreen = ({ navigation }: Props) => {
 
   const assertValidInput = (): boolean => {
     if (input.length <= 2) {
-      setMessage(translate("UsernameScreen.3CaractersMinimum"))
-      setMessageIsError(true)
+      if (!init) {
```

Comment:
I wouldn't expect this function to have a side effect. Can we decouple the logic of `validness` from the resulting behaviour (error message + focus)?